packages/amplify-util-mock/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@aws-amplify/amplify-appsync-simulator": "^2.11.0",
     "@aws-amplify/amplify-category-function": "^5.2.0",
-    "@aws-amplify/amplify-provider-awscloudformation": "^8.3.0-aws-cdk-lib-2-28.0",
+    "@aws-amplify/amplify-provider-awscloudformation": "^8.4.0-rc.43f73ab268.0",
```

Comment:
Is cdk version change also included in the RC?

Reply:
Both are at `~2.80.0`.